packages/office-ui-fabric-react/src/components/SwatchColorPicker/ColorPickerGridCell.styles.ts

```diff
@@ -72,14 +77,26 @@ export const getStyles = (props: IColorPickerGridCellStyleProps): IColorPickerGr
       },
       selected && {
         padding: DIVIDING_PADDING,
-        border: `${calculatedBorderWidth}px solid ${theme.palette.neutralTertiaryAlt}`
+        border: `${calculatedBorderWidth}px solid ${buttonBorderChecked}`,
+        selectors: {
+          ['&:hover::before']: {
+            content: '""',
+            height: height,
+            width: width,
+            position: 'absolute',
+            top: -calculatedBorderWidth,
+            left: -calculatedBorderWidth,
+            borderRadius: circle ? '50%' : 'default',
+            boxShadow: `inset 0 0 0 1px ${buttonBorderCheckedHovered}`
```

Comment:
Love it how you used the `boxShadow` here. Nice job 👍

Reply:
Thanks!